documentation/book/con-product-downloads.adoc

```diff
@@ -3,14 +3,24 @@
 // getting-started.adoc
 
 [id='downloads-{context}']
-= {ProductName} downloads
+= Installing {ProductName} and deploying components
 
 ifdef::Downloading[]
-{ProductName} releases are available to download from {ReleaseDownload}. The release artefacts contain documentation, installation, and example `.yaml` files for deployment on {ProductPlatformName}. The installation, and example files are used throughout this documentation. Additionally, a Helm Chart is provided for deploying the Cluster Operator using link:https://helm.sh/[Helm^]. The container images are available through the {DockerRepository}.
+To install {ProductName}, download the release artefacts from {ReleaseDownload}.
 endif::Downloading[]
+
 ifndef::Downloading[]
-The following procedures use installation and example YAML files to help you install {ProductName} on {ProductPlatformName}, perform common operations, and configure your Kafka cluster. {ExampleYAMLURL} are available to download from the AMQ Streams website.
+To install {ProductName}, download and extract the `install_and_examples.zip` file from the {ZipDownload}.
+endif::Downloading[]
 
-NOTE: Container images for {ProductName} are available in the {ReleaseDownload}. We recommend that you use the installation and example YAML files instead of downloading the container images.
+The folder contains several YAML files to help you deploy the components of {ProductName} to {ProductPlatformName}, perform common operations, and configure your Kafka cluster. The YAML files are referenced throughout this documentation.
 
+ifdef::Downloading[]
+Additionally, a Helm Chart is provided for deploying the Cluster Operator using link:https://helm.sh/[Helm^]. The container images are available through the {DockerRepository}.
+endif::Downloading[]
+
+The remainder of this chapter provides an overview of each component and instructions for deploying the components to {ProductPlatformName} using the YAML files provided.
+
+ifndef::Downloading[]
+NOTE: Although container images for {ProductName} are available in the {ReleaseDownload}, we recommend that you use the YAML files provided instead.
```

Comment:
I do not think this note applies only to downstream. It should use `{DockerRepository}` as the link to the repository and it can be showed everywhere.

Reply:
Alright, I'll change that.